Explorer/Assets/DCL/Landscape/TerrainFactory.cs

```diff
@@ -72,6 +72,7 @@ public Collider CreateBorderCollider(string name, Transform parent, Vector3 size
         {
             BoxCollider collider = new GameObject(name).AddComponent<BoxCollider>();
             collider.transform.SetParent(parent);
+            collider.gameObject.layer = LayerMask.NameToLayer("InvisibleColliders");
```

Comment:
Maybe we can create a class with some constants like we are doing with the hashes of animators

Reply:
would be nice, but maybe in more calm times.